x-pack/test/performance/constants.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export enum Journey {
```

Comment:
Is it possible that we don't need this enum and instead infer the journey name from the directory name instead? Looking below, the pattern is always `$SOME_DIRECTORY/$JOURNEY_NAME/config.ts`

Reply:
Even though I don't have any strong opinions on this one and it seems more convenient for now, I'd prefer to keep the journeys to be specified explicity, there is two major reasons that comes to my mind:
- We don't know how developers are going to structure their journey folders & files, inferring from folder name feels like a little bit we are pushing them to use certain organization.
- We will probably want to update [kbn-test](https://github.com/elastic/kibana/tree/main/packages/kbn-test#:~:text=The%20%40kbn/test%20package%20provides%20ways%20to%20run%20tests.%20Currently%20only%20functional%20testing%20is%20provided%20by%20this%20library%2C%20with%20unit%20and%20other%20testing%20possibly%20added%20here.) with performance journeys or we might wanna have some other test runner specific to performance journeys so in that case folder and config structure will change. (It feels like I'm saying we are going to change it anyways in future let's not do it now but I'm not completely opposed to your idea 😅)